ballista/rust/core/src/serde/logical_plan/to_proto.rs

```diff
@@ -407,6 +407,9 @@ impl From<&DataType> for protobuf::arrow_type::ArrowTypeEnum {
                     fractional: *fractional as u64,
                 })
             }
+            DataType::Map(_, _) => {
+                unimplemented!("Ballista does not yet support Map data type")
```

Comment:
I am not enough of an expert to implement protobuf serialization in Ballista for a new `DataType` at this time but I suspect it is not very hard